crates/electrum/tests/test_electrum.rs

```diff
@@ -22,10 +23,34 @@ fn get_balance(
     Ok(balance)
 }
 
+fn electrum_sync<I>(
+    client: &BdkElectrumClient<Client>,
+    spks: &[ScriptBuf],
```

Comment:
**Nit:** there is no point having a reference here, since we are cloning this later on (with `.to_owned()`).